server/bin/www

```diff
@@ -87,3 +90,33 @@ function onListening() {
 
   console.log(`Listening on ${bind}`.red.underline.bold);
 }
+
+/**
+ * Authentication and listener for user connection to socket
+ */
+
+io.use(async (socket, next) => {
+  const token = socket.handshake.headers.cookie
+
+  if (!token) {
+    console.log("Invalid token")
+  }
```

Comment:
Let's get rid of codes used for debugging

Reply:
Forgot to convert these to errors